.travis/rebuild_image.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 REPOSITORY=${REPOSITORY:-nefelinetworks/bess_build}
 VERSION=${VERSION:-`date +%y%m%d`}
-docker build --build-arg BESS_DPDK_BRANCH=${BESS_DPDK_BRANCH:-develop} -t $REPOSITORY:latest -t $REPOSITORY:$VERSION .
+docker build --build-arg BESS_DPDK_BRANCH=${BESS_DPDK_BRANCH:-c++} -t $REPOSITORY:latest -t $REPOSITORY:$VERSION .
```

Comment:
This script broke due to a previous change to the develop branch. We should keep it working.
Also: perhaps we won't be renaming c++ to develop.